### ISSUE TITLE
Attempting to kill Discord

### ISSUE THREAD

**Issue by HarryplaysOMG4** [BetterDiscord/Installer#316]
**Describe the bug**
stuck on Attempting to kill Discord. should i close betterdiscord installer? i sitll have it out

**To Reproduce**
1. Open betterdiscord installer
2. do the things before starting install
3. the install begins, reach ✅ Shims injected
4. see Attempting to kill Discord error

**Expected behavior**
im pretty sure it was gonna restart discord and close the installer or something.

**Screenshots**
![image](https://user-images.githubusercontent.com/112706650/218337360-c5902423-db25-4e61-ae98-30cda3a1c660.png)

**System Information:**
 - OS: Windows 7 ultimate X32 system
- Discord Version: app-1.0.9010

**Additional context**
I closed discord and reopened it about 3 times now. Still not fixed

**Comment by ShadowDevilsAvenged** [comment]
Redo the installer until it stops at "Killing Discord" then without closing the installer, restart your pc, then check if bd got installed

**Comment by PurpleBored** [comment]
i had the same bug it seems Windows 7 related i just killed discord manualy using task manager and then close BD installer then it worked for me

**Comment by zerebos** [comment]
This is a problem with one of our dependencies on Windows 7, we are looking into workarounds.

**Comment by zerebos** [comment]
Duplicate of #287

**Comment by zer0origin** [comment]
Hi, I  still get this  issue on windows 10 using the build [v1.2.1](https://github.com/BetterDiscord/Installer/releases/tag/v1.2.1)

Does anyone know any possible reasons for this? My discord was updated sometime near the end of 2022; after BetterDiscord no longer worked. Reinstall attempts of both Discord and BetterDiscord failed (including manually cleaning up the uninstall), using the new release has not helped; neither has using previous ones, I've tried having discord closed, closing discord during the attempt and having it open but nothing seems to work.

I search online and I could only find a little information about checking my PATH variable, which was fine. So Im out of options.

please, let me know if anyone has any idea how to fix the issue.

**Comment by zerebos** [comment]
Your best bet is joining our Discord support server where we can help you in a more active setting than GitHub
